recipe/meta.yaml

```diff
@@ -23,6 +23,7 @@ requirements:
     - nasm
     - make
     - python >=2.7,<3
+    - perl >=5.8.0
```

Comment:
It seems like perl is maybe a host dep, if it is also a run dep

Reply:
That may definitely be the case, still haven't fully understood the host vs build distinction

Reply:
You can think of it as a cross-compiler distinction.  `build` is the architecture of the machine you are actually building on (e.g. linux-64), where as `host` is the target architecture that you are building for and so it is arch where dynamic libraries are needed to link against (e.g. osx-64). 

So compilers and build tools tend to go in build, while libraries that the code uses go into host.

Reply:
perl is in a werid place because it is often used as a build tool, but is also a target to be linked against

Reply:
Ah, I was getting held up on the Python example which is being used as a build tool. I need to think of Perl as being a lib that I'm linking against.